### PR TITLE
Write some READMEs for VPA

### PR DESCRIPTION
**Pull request by tkulczynski**
cc @kgrygiel

### CODE REVIEW

File: vertical-pod-autoscaler/README.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    updateMode: "Auto"
+```
+
+### Known limitations
+
+* The VPA admission controller is an admission webhook. The order of admission
+  controllers is defined by flag on APIserver.
+  VPA admission controller might have **relations to other admission controllers**,
+  e.g. running it after quota admission might cause making incorrect decisions.
+* Out-of-memory events / pod evictions are not taken into account for memory
```
Comment (mwielgus) [inline vertical-pod-autoscaler/README.md]: not taken YET

File: vertical-pod-autoscaler/README.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  name: my-app-vpa
+spec:
+  selector:
+    matchLabels:
+      app: my-app
+  updatePolicy:
+    updateMode: "Auto"
+```
+
+### Known limitations
```
Comment (mwielgus) [inline vertical-pod-autoscaler/README.md]: Limitations of the alpha versionion

File: vertical-pod-autoscaler/README.md (hunk 1 above)
Comment (tkulczynski) [reply vertical-pod-autoscaler/README.md]: Done

File: vertical-pod-autoscaler/README.md (hunk 2 above)
Comment (tkulczynski) [reply vertical-pod-autoscaler/README.md]: Done

**Pull request merged**